### DISCUSSION TITLE
[Q&A] 自部署时 Error 526 导致无法流式返回

### DISCUSSION THREAD

**Discussion by itq5** [lobehub/lobehub#540] [Self Hosting] [answered]
### 💻 系统环境 | Operating System

Windows

### 🌐 浏览器 | Browser

Chrome

### 🐛 问题描述 | Bug Description

从oneapi对接到该项目，不支持流式显示吗？我换了很多模型都是直接全量显示，不是一段一段显示出来的

### 🚦 期望结果 | Expected Behavior

流式显示，一段一段打印出来

### 📷 复现步骤 | Recurrence Steps

对接one api ,不对接场景没测试

**Comment by mushan0x0** [comment]
https://chat-preview.lobehub.com/ 在这个预览页面填上你的服务看看，这个应该是部署的问题

[a comment by an automated account is not shown]

**Comment by itq5** [comment]
> https://chat-preview.lobehub.com/ 在这个预览页面填上你的服务看看，这个应该是部署的问题

```json
{
  "endpoint": "http://xxxxxxx/",
  "error": {
    "headers": {
      "cache-control": "private, max-age=0, no-store, no-cache, must-revalidate, post-check=0, pre-check=0",
      "cf-as-number": "16509",
      "cf-ray": "82e0c1d5b6288b57-HKG",
      "connection": "keep-alive",
      "content-length": "15",
      "content-type": "text/plain; charset=UTF-8",
      "date": "Thu, 30 Nov 2023 05:40:42 GMT",
      "expires": "Thu, 01 Jan 1970 00:00:01 GMT",
      "referrer-policy": "same-origin",
      "server": "cloudflare",
      "x-frame-options": "SAMEORIGIN"
    },
    "stack": "Error: 526 error code: 526\n    at (vc/edge/function:135:38921)\n    at (vc/edge/function:138:6188)\n    at (vc/edge/function:138:6938)\n    at (vc/edge/function:123:1773)\n    at (vc/edge/function:127:85995)",
    "status": 526
  }
}
```

会报错。

[a comment by an automated account is not shown]

**Comment by mushan0x0** [comment]
多半是加了 cloudflare 没配置好导致的
<img width="778" alt="image" src="https://github.com/lobehub/lobe-chat/assets/29084441/3de35b3e-94dd-4463-a270-be543e152893">

**Comment by itq5** [comment]
> https://chat-preview.lobehub.com/

会不会可能是https://chat-preview.lobehub.com/网站的防护？我尝试对接one api以及上游api都是一样的526报错

[a comment by an automated account is not shown]

**Comment by binaryYuki** [comment, marked as answer]
> > https://chat-preview.lobehub.com/
> 
> 会不会可能是[https://chat-preview.lobehub.com/网站的防护？我尝试对接one](https://chat-preview.lobehub.com/%E7%BD%91%E7%AB%99%E7%9A%84%E9%98%B2%E6%8A%A4%EF%BC%9F%E6%88%91%E5%B0%9D%E8%AF%95%E5%AF%B9%E6%8E%A5one) api以及上游api都是一样的526报错

@itq5 

One-api对接报错526是cloudflare的问题 
要是是自家服务器：
- 尝试renew你的ssl证书
- 在cloudflare的log里面检查错误原因
- 详见 [cloudflare docs](https://community.cloudflare.com/t/community-tip-fixing-error-526-invalid-ssl-certificates/44273)

要是是第三方部署请提供部署平台名

**Comment by itq5** [comment]
已经解决，是我的反向代理问题，感谢支持！

**Reply by flyweb291** [reply]
怎么解决的请问

[a comment by an automated account is not shown]

**Comment by pzcn** [comment]
可以尝试在nginx配置文件中添加下面的内容
```proxy_cache off;  # 关闭缓存
proxy_buffering off;  # 关闭代理缓冲
chunked_transfer_encoding on;  # 开启分块传输编码
tcp_nopush on;  # 开启TCP NOPUSH选项，禁止Nagle算法
tcp_nodelay on;  # 开启TCP NODELAY选项，禁止延迟ACK算法
keepalive_timeout 300;  # 设定keep-alive超时时间为65秒
